plugins/telink-dfu/fu-telink-dfu-hid-device.c

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright 2024 Mike Chang <Mike.chang@telink-semi.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ */
+
+#include "config.h"
+
+#include <linux/hidraw.h>
+#include <linux/input.h>
```

Comment:
do you need these two includes?

Reply:
resolved in commit 0fb74d587e8aedf6a6bccabff38623ea034d224d
removed the includes